packages/virtual/emulators/package.mk

```diff
@@ -128,7 +128,16 @@ makeinstall_target() {
   esac
 
   ### Commodore Amiga
-  add_emu_core amiga retroarch puae true
+  case ${DEVICE} in
+    RK35*|RK3326|RK3399)
+      add_emu_core amiga retroarch puae2021 true
+      add_emu_core amiga retroarch puae false
+    ;;
+    *)
+      add_emu_core amiga retroarch puae true
+      add_emu_core amiga retroarch puae2021 false
```

Comment:
Did we test puae2021 on non-ARM devices?  If not, no worries we can test after the merge.

Reply:
Unfortunately I have no other devices to test Jelos on than rg503

Reply:
No worries, I tested on AMD64 and x55 and it's working fine there.  I'll test on other devices when I have a sec.  Nice work on this! 🤘